bin/templates/project/app/build.gradle

```
@@ -291,6 +300,11 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: '*.jar')
+
+    if (cdvHelpers.getConfigPreference('EnableKotlin', 'false').toBoolean()) {
+        compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Why do we require the `jdk7` stdlib here? This should be changed to `kotlin-stdlib` instead.

```suggestion
        compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```